src/collision/collision_regime.f90

```diff
@@ -364,7 +372,7 @@ function calc_theta(r_tar, v_tar, r_imp, v_imp) result(theta)
             x_cross_v(:) = distance(:) .cross. imp_vel(:) 
             sintheta = norm2(x_cross_v(:)) / norm2(distance(:)) / norm2(imp_vel(:))
 
-            theta = asin(sintheta) ! Find a more exact way to calculate theta
+            theta = (PI / 2) - abs(asin(sintheta)) ! Find a more exact way to calculate theta
```

Comment:
What does the comment here mean? 
> Find a more exact way to calculate theta

Reply:
Edited the comment to reflect the calculation. The angle is being calculated correctly with respect to the horizon.